hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -115,25 +115,27 @@
 @SuppressWarnings({"unused", "SameParameterValue", "WeakerAccess"})
 public abstract class HazelcastTestSupport {
 
-    private static final boolean EXPECT_DIFFERENT_HASHCODES = (new Object().hashCode() != new Object().hashCode());
-
     public static final String JAVA_VERSION = System.getProperty("java.version");
     public static final String JVM_NAME = System.getProperty("java.vm.name");
+
     public static final int ASSERT_TRUE_EVENTUALLY_TIMEOUT;
 
+    private static final boolean EXPECT_DIFFERENT_HASHCODES = (new Object().hashCode() != new Object().hashCode());
+    private static final ILogger LOGGER = Logger.getLogger(HazelcastTestSupport.class);
+
     @Rule
     public JitterRule jitterRule = new JitterRule();
 
     @Rule
     public DumpBuildInfoOnFailureRule dumpInfoRule = new DumpBuildInfoOnFailureRule();
 
+    private TestHazelcastInstanceFactory factory;
+
     static {
         ASSERT_TRUE_EVENTUALLY_TIMEOUT = getInteger("hazelcast.assertTrueEventually.timeout", 120);
-        System.out.println("ASSERT_TRUE_EVENTUALLY_TIMEOUT = " + ASSERT_TRUE_EVENTUALLY_TIMEOUT);
+        LOGGER.fine("ASSERT_TRUE_EVENTUALLY_TIMEOUT = " + ASSERT_TRUE_EVENTUALLY_TIMEOUT);
```

Comment:
This `System.out.println` always gets printed out in the middle of other logs or its trailing `newline` is swallowed (so it's merged into another line). And I think it's being read once in a 100 years, so making it less visible seems to be a good solution to me.